Modules/Loadable/Markups/MRML/vtkMRMLMarkupsJsonStorageNode.cxx

```diff
@@ -52,11 +52,13 @@
 
 namespace
 {
+  // TODO: For now we leave schema GitHub branch as "master" to maintain compatibility with Slicer < 5.1.
```

Comment:
```suggestion
  // Schema ID used to be an URL where the schema was available, but since "master" branch was renamed
  // to "main" the URL does not resolve to a valid file anymore. This is not a problem, because Schema ID does
  // not have to correspond to an URL where the schema is available according to the JSON standard, it is just 
  // recommended for compatibility with software that assumes this. The ID will likely to be changed to have
  // "main" in the name in the future, but for compatibility with Slicer < 5.1 the current value is preserved for now.
```